Lib/fontTools/merge.py

```diff
@@ -780,6 +849,7 @@ def merge(self, fontfiles):
 		# TODO Is it necessary to reload font?  I think it is.  At least
 		# it's safer, in case tables were loaded to provide glyph names.
 		fonts = [ttLib.TTFont(fontfile) for fontfile in fontfiles]
+		self.fonts = copy.deepcopy(fonts)
```

Comment:
We should have a drink one day.. Yeah changing the variable name is annoying since it's hard for us to track its usage.. I faced the same problem before..